src/wp/SplashScreen.cs

```diff
@@ -40,26 +40,35 @@ namespace WPCordovaClassLib.Cordova.Commands
     /// </summary>
     public class SplashScreen : BaseCommand
     {
+        private readonly Preferences preferences = new Preferences();
         private Popup popup;
-        private bool autohide = true;
 
         private static bool WasShown = false;
 
         public SplashScreen()
         {
-            Image SplashScreen = new Image();
-            BitmapImage splash_image = new BitmapImage();
-            splash_image.SetSource(Application.GetResourceStream(new Uri(@"SplashScreenImage.jpg", UriKind.Relative)).Stream);
-            SplashScreen.Source = splash_image;
+            this.LoadPreferencesFromConfiguration();
+
+            Image SplashScreen = new Image()
+            {
+                Height = Application.Current.Host.Content.ActualHeight,
+                Width = Application.Current.Host.Content.ActualWidth,
+                Stretch = Stretch.Fill
+            };
+
+            var imageResource = Application.GetResourceStream(preferences.SplashScreenImage);
+            if (imageResource != null)
+            {
+                BitmapImage splash_image = new BitmapImage();
+                splash_image.SetSource(imageResource.Stream);
+                SplashScreen.Source = splash_image;
+            }
 
             // Instansiate the popup and set the Child property of Popup to SplashScreen
-            popup = new Popup() {IsOpen = false, Child = SplashScreen };
+            this.popup = new Popup() { IsOpen = false, Child = SplashScreen };
```

Comment:
Nit: prefixed use of "this" when referring to members isn't bad, but is unnecessary.